addons/dotnet/dotnet.addon

```diff
@@ -0,0 +1,9 @@
+# Name: dotnet
+# Description: ImageStreams and Templates for developing and deploying application using .NET Core
+# URL: https://github.com/redhat-developer/s2i-dotnetcore
+
+oc apply --namespace openshift -f dotnet_imagestreams.json
```

Comment:
@LalatenduMohanty https://github.com/redhat-developer/s2i-dotnetcore/blob/master/dotnet_imagestreams_centos.json what about this as part of imagestream for upstream?

Reply:
@praveenkumar good point. Will check.